### ISSUE TITLE
Problem when starting Twilight Menu ++ on DSi XL.

### ISSUE THREAD

**Issue by Jonas195** [DS-Homebrew/TWiLightMenu#2307]
**Flashcard or Exploits? *Sudoku* 
**Base Console:*DSi XL - 1.4.5U* 

**TWiLight Menu++ Version: *v26.1.1 and ealier* 
**TWiLight Menu++ Theme:  *Nintendo DSi* 

### Steps to reproduce

1. Open twilight menu
2. Open a Game
3. Restart the Ds
4. Enter Twilight again.

**Comment by RocketRobz** [comment]
If it's a Guru Meditation Error, then idk why this happens.
Open `sd:/_nds/TWiLightMenu/settings.ini`, then set `LOGGING` to `1`. When the error occurs again, please upload the log file from `sd:/_nds/TWiLightMenu/`.

**Comment by Jonas195** [comment]
Yes, it's a Guru Meditation Error. I've already tried changing the logging to 1, but I don't know which log file to look for. In any case, Twilight didn't start up.

**Comment by RocketRobz** [comment]
> Yes, it's a Guru Meditation Error. I've already tried changing the logging to 1, but I don't know which log file to look for. In any case, Twilight didn't start up.

The file should be called `log.txt` or smth.

**Comment by Jonas195** [comment]
I don't have any files with those names :c

**Comment by RocketRobz** [comment]
Does this still occur on later versions?
